server.js

```diff
@@ -1,299 +1,299 @@
-// TASK 1: Setting up server
-
-
-const http = require('http');
-
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
-
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
-
-server.listen(PORT, HOSTNAME, () => {
-  console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-});
-
-
-
-
-// Task 2 : Connecting with mongo db data base 
-
-/*
-const http = require('http');
-const { MongoClient } = require('mongodb');
-
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
-
-const mongoUri = 'mongodb+srv://<user_name>:<password>s@<mongo_cluster_url>';
-const client = new MongoClient(mongoUri, { useNewUrlParser: true, useUnifiedTopology: true });
-
-async function startServer() {
-  try {
-    // Connect to MongoDB
-    await client.connect();
-    console.log('Connected to MongoDB');
-
-    // Create HTTP Server
-    const server = http.createServer((req, res) => {
-      res.statusCode = 200;
-      res.setHeader('Content-Type', 'text/plain');
-      res.end('Hello World\n');
-    });
-
-    // Start HTTP Server
-    server.listen(PORT, HOSTNAME, () => {
-      console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-    });
-  } catch (err) {
-    console.error('Could not connect to MongoDB:', err);
-    process.exit(1);
-  }
-}
-
-startServer();
-
-*/
-
-
-
-
-// TASK 3 : Use mongoose instead of mongodb driver 
-
-/**
-const http = require('http');
-const mongoose = require('mongoose');
-
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
-
-const uri = "mongodb+srv://suraj_admin:suraj_admin@cluster0.dme40pl.mongodb.net/?retryWrites=true&w=majority";
-
-async function startServer() {
-  try {
-    // Connect to MongoDB using Mongoose
-    await mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
-    console.log('Connected to MongoDB');
-
-    // Create HTTP Server
-    const server = http.createServer((req, res) => {
-      res.statusCode = 200;
-      res.setHeader('Content-Type', 'text/plain');
-      res.end('Hello World\n');
-    });
-
-    // Start HTTP Server
-    server.listen(PORT, HOSTNAME, () => {
-      console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
-    });
-  } catch (err) {
-    console.error('Could not connect to MongoDB:', err);
-    process.exit(1);
-  }
-}
-
-startServer();
-
- */
-
-
-
```

Comment:
The code for Task 3 is commented out. Please remove the commented code to ensure the code can be executed properly.